.github/workflows/main.yml

```diff
@@ -47,26 +47,10 @@ jobs:
       - name: Use latest esy
         run: npm install --global esy
 
-      # This step is required because esy's home directory doesn't exist at this point
       - name: Install esy packages
```

Comment:
This step should probably be renamed since its doing more than just installing esy packages.

Reply:
I agree with part of it, but it's actually the same as what opam does except for the cache part.